front-end/src/components/Spinner.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const useStyles = makeStyles(() => ({
+  spinner: {
+    margin: 'auto',
+    display: 'block',
+    width: '120px',
+    height: '120px',
+  },
+}));
+const Spinner = () => {
```

Comment:
`const Spinner = ({ className }) => {`

className Prop to receive over Styling from Parent

Reply:
added ;)